src/modules/chat_history/index.js

```diff
@@ -0,0 +1,74 @@
+const $ = require('jquery');
+const watcher = require('../../watcher');
+const keyCodes = require('../../utils/keycodes');
+
+const CHAT_INPUT_SELECTOR = '.chat-input textarea';
+const AUTOCOMPLETE_SUGGESTIONS_SELECTOR = '[data-a-target=autocomplete-balloon]';
+
+function isSuggestionsShowing() {
+    return $(AUTOCOMPLETE_SUGGESTIONS_SELECTOR).length > 0;
+}
+
+function setTextareaValue($inputField, msg) {
```

Comment:
If you're wondering why I use this, it's because without a 'input' event, the textarea value doesn't stick (like if someone post a message in the chat). This is what's happening currently in the `DoubleClickMentionModule`.